src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -260,6 +261,9 @@ class TezosNodeOperator(val node: TezosRPCInterface, val network: String, batchC
     (fetchProposals, fetchBakers, fetchBallots).tupled.run(blocks.filterNot(b => isGenesis(b.data)))
   }
 
+  //move it to the node operator
+  def getDelegatesForBlock(keys: List[PublicKeyHash], blockHash: BlockHash): Future[Map[Delegate, Contract]] = ???
```

Comment:
is that needed?

Reply:
Must've been a leftover of ongoing work... I'll remove it and see if anything breaks, thanks